handler/handler_test.go

```diff
@@ -25,19 +25,24 @@ func TestServeHTTP(t *testing.T) {
 		expLastModified bool
 	}{
 		{
-			desc: "should replace foo by bar",
+			desc:            "should replace foo by bar",
+			contentEncoding: "",
+			contentType:     "text/html",
 			rewrites: []Rewrite{
 				{
 					Regex:       "foo",
 					Replacement: "bar",
 				},
 			},
-			contentType: "text/html",
-			resBody:     "foo is the new bar",
-			expResBody:  "bar is the new bar",
+			lastModified:    false,
+			resBody:         "foo is the new bar",
+			expResBody:      "bar is the new bar",
+			expLastModified: false,
```

Comment:
instead of modifying these tests, please duplicate them with the updated test parameters. I know it seems redundant but we need to ensure updates do not create breaking changes.